src/03-modality-agnostic-files.md

```diff
@@ -217,3 +217,198 @@ code organization of these scripts at the moment.
+ 1. For datasets and derivatives, provenance can also include details of why the data were collected in the first place covering hypotheses, claims, and prior publications. Provenance can encode support for which claims were supported by future analyses.
+ 1. Provenance can involve information about people and institutions involved in a study.
+ 1. Provenance records can highlight reuse of datasets while providing  appropriate attribution to the original dataset generators as well as future transformers.  
+
+Provenance can be captured using different mechanisms, but independent of encoding, always reflects transformations by either humans or software. The interpretability of provenance records requires a consistent vocabulary for provenance as well as an expectation for a consistent terminology for the objects being encoded. 
+
+Encoding Provenance In BIDS
+
+i. Provenance information SHOULD be included in a BIDS dataset when possible.
+
+ii. Provenance records MUST use the [PROV model](https://www.w3.org/TR/prov-o/) ontology and SHOULD be augmented by terms curated in the BIDS specification, the [NIDM](http://nidm.nidash.org/) model, and future enhancements to these models.
+
+iii. If provenance records are included, these records of provenance of a dataset or a file MUST be described using a `[&lt;prefix>_]prov.jsonld` file. Since these [jsonld](https://json-ld.org/) documents are graph objects, they can be aggregated without the need to apply any inheritance principle. 
+
+iv. The provenance file MAY be used to reflect the _provenance of a dataset, a collection of files or a specific file at any level_of the bids hierarchy. 
+
+v. Provenance information SHOULD be anonymized/de-identified as necessary. 
+
+Justification for Separating Provenance from file JSON
+
+Provenance is information about a file, including any metadata that is relevant to the file itself. Thus any BIDS data file and its associated JSON sidecar metadata together constitute a unique entity. As such, one may want to record the provenance of the JSON file as much as the provenance of the BIDS file. In addition, separating the provenance as a separate file for now, allows this to be an OPTIONAL component, and by encoding provenance as a JSON-LD document allows capturing the provenance as an individual record or multiple records distributed throughout the dataset.
+
+Possible places to encode provenance
+
+**Dataset level provenance.** At the dataset level, provenance could be about the dataset itself, or about any entity in the dataset. This provenance may evolve as new data are added, which may include sourcedata, BIDS data, and BIDS derived data. One option is to make use of <code>[https://w3c.github.io/json-ld-syntax/#named-graphs](https://w3c.github.io/json-ld-syntax/#named-graphs)</code>
```

Comment:
Why do you want to put your `[https://w3c.github.io/json-ld-syntax/#named-graphs]` link into a code block? If you are certain that that's necessary, you can use the backtick sign: `

once at the beginning and once at the end for inline code: `like so`

assuming you just want the link DESCRIPTION to be formatted like code, you need to put the backticks *inside* the square brackets: [`https://w3c.github.io/json-ld-syntax/#named-graphs`](https://w3c.github.io/json-ld-syntax/#named-graphs)

[ + backtick + description + backtick + ] + ( + url + )

see: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code

Reply:
interestingly, this works in the "preview" of the comment I just posted. But looking at the rendering now, it seems that the link description wrapped by backticks does NOT look like a codeblock ... meh :man_shrugging: 

pic of the preview:

![image](https://user-images.githubusercontent.com/9084751/77767001-b7144c00-7040-11ea-9b1f-0ee7655659f4.png)

Reply:
@sappelhoff OK you are right, fixing this

Reply:
mh okay, this didn't fix linkchecker. Seems more like a linkchecker issue ... your link works :man_shrugging:

Reply:
discussed with @cmaumet , i'll check by only keeping the link between the brackets, like :
```markdown
[named-graphs](https://w3c.github.io/json-ld-syntax/#named-graphs)
```